contracts/src/v0.8/ccip/pools/GHO/UpgradeableTokenPool.sol

```diff
@@ -317,4 +328,21 @@ abstract contract UpgradeableTokenPool is IPool, OwnerIsCreator, IERC165 {
     if (IARM(i_armProxy).isCursed()) revert BadARMSignal();
     _;
   }
+
+  /// @notice Getter for proxy pool address.
+  /// @return proxyPool The proxy pool address for the given remoteChainSelector
+  function getProxyPool() public view returns (address proxyPool) {
```

Comment:
Similar to my other feedback, this should be `getPoolProxy`

Reply:
naming is aligned with CCIP terms, but `return` docs is outdated @DhairyaSethi

Reply:
[9dceff0](https://github.com/aave/ccip/pull/18/commits/9dceff0ec9259cb3c0eafb978168ceb1b89adbcf)